Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleGeoTrackingSceneView.swift

```diff
@@ -0,0 +1,300 @@
+                        
+                        sceneViewProxy.updateCamera(
+                            frame: frame,
+                            cameraController: cameraController,
+                            orientation: interfaceOrientation,
+                            initialTransformation: .identity
+                        )
+                        sceneViewProxy.setFieldOfView(
+                            for: frame,
+                            orientation: interfaceOrientation
+                        )
+                    }
+                
+                if initialCameraIsSet {
+                    sceneViewBuilder(sceneViewProxy)
+                        .cameraController(cameraController)
+                        .attributionBarHidden(true)
+                        .spaceEffect(.transparent)
+                        .atmosphereEffect(.off)
+                        .interactiveNavigationDisabled(true)
+                        .onCameraChanged { camera in
+                            currentCamera = camera
+                        }
+                }
+                
+                if !coachingOverlayIsHidden {
+                    ARCoachingOverlay(goal: .geoTracking)
+                        .sessionProvider(arViewProxy)
+                        .active(coachingOverlayIsActive)
+                        .allowsHitTesting(false)
+                }
+            }
+            .observingInterfaceOrientation($interfaceOrientation)
+            .onAppear {
+                arViewProxy.session.run(configuration)
+            }
+            .onDisappear {
+                arViewProxy.session.pause()
+                Task { await locationDataSource.stop() }
+            }
+            .task {
+                do {
+                    try await locationDataSource.start()
+                    await withTaskGroup(of: Void.self) { group in
+                        group.addTask {
+                            for await location in locationDataSource.locations {
+                                lastLocationTimestamp = location.timestamp
+                                currentLocation = location
+                                await updateSceneView(for: location)
+                            }
+                        }
+                        group.addTask {
+                            for await heading in locationDataSource.headings {
+                                currentHeading = heading
+                                
+                                guard let currentCamera else { return }
+                                let camera = cameraController.originCamera.rotatedTo(
+                                    heading: calibrationHeading ?? heading,
+                                    pitch: currentCamera.pitch,
+                                    roll: currentCamera.roll
+                                )
+                                sceneViewProxy.setViewpointCamera(camera)
```

Comment:
After making the heading updates come through, in my testing the `setViewpointCamera` method is interfering with the frame update. We can discuss and test different ideas of updating the camera smoothly.

I think the questions we need to figure out are…

1. In the past we only use this method to adjust camera when **not** using `ARKit`. That is not the case for our new design right? If we always use ARKit now, then it already does the tracking for us.
2. When using ARKit, it obtains the location and heading from the device's GPS. If we instead using a different LDS such as NMEALDS, then it may make sense to align the location and heading between ARKit and the custom LDS.
3. For initial mode, once the initial location and heading are set, and the session starts, later heading adjustments rely on the calibration view.

So IMO we can leave this out for now, and figure it out once we have a clear separation between `initial` and `continuous` modes.

Reply:
Okay, I will revert the last commit that uses `setViewpointCamera` and it can be added later if necessary.